docs/communication_protocols/dronecan_protocol.rst

```diff
@@ -434,6 +448,89 @@ Stow Result
 This parameter is available on modules that support :ref:`manual_stow_position`. It reports on how the previous stow attempt ended. See the
 :ref:`stow_result` section for more information.
 
+.. note::
+	The following parameters are not yet available on all modules that support DroneCAN. If you do not see these parameters, check `vertiq.co <https://www.vertiq.co/>`_ for a new firmware release for 
+	your module.
```

Comment:
This may not be quite complete advice at the moment considering no releases have these new parameters on them yet. You may want to add something like "If you are on the latest release and do not have access to these parameters, future updates for your module may add support for these messages".

Reply:
ok.